src/view/com/profile/ProfileMenu.tsx

```diff
@@ -0,0 +1,274 @@
+        }
+      }
+    }
+  }, [profile.viewer?.muted, track, queueUnmute, _, queueMute])
+
+  const onPressBlockAccount = React.useCallback(async () => {
+    if (profile.viewer?.blocking) {
+      track('ProfileHeader:UnblockAccountButtonClicked')
+      openModal({
+        name: 'confirm',
+        title: _(msg`Unblock Account`),
+        message: _(
+          msg`The account will be able to interact with you after unblocking.`,
+        ),
+        onPressConfirm: async () => {
+          try {
+            await queueUnblock()
+            Toast.show(_(msg`Account unblocked`))
+          } catch (e: any) {
+            if (e?.name !== 'AbortError') {
+              logger.error('Failed to unblock account', {message: e})
+              Toast.show(_(msg`There was an issue! ${e.toString()}`))
+            }
+          }
+        },
+      })
+    } else {
+      track('ProfileHeader:BlockAccountButtonClicked')
+      openModal({
+        name: 'confirm',
+        title: _(msg`Block Account`),
+        message: _(
+          msg`Blocked accounts cannot reply in your threads, mention you, or otherwise interact with you.`,
+        ),
+        onPressConfirm: async () => {
+          try {
+            await queueBlock()
+            Toast.show(_(msg`Account blocked`))
+          } catch (e: any) {
+            if (e?.name !== 'AbortError') {
+              logger.error('Failed to block account', {message: e})
+              Toast.show(_(msg`There was an issue! ${e.toString()}`))
+            }
+          }
+        },
+      })
+    }
+  }, [profile.viewer?.blocking, track, openModal, _, queueUnblock, queueBlock])
+
+  const onPressReportAccount = React.useCallback(() => {
+    track('ProfileHeader:ReportAccountButtonClicked')
+    openModal({
+      name: 'report',
+      did: profile.did,
+    })
+  }, [track, openModal, profile])
+
+  return (
+    <EventStopper onKeyDown={false}>
+      <Menu.Root>
+        <Menu.Trigger label={_(`More options`)}>
+          {({props}) => {
+            return (
+              <TouchableOpacity
+                {...props}
+                hitSlop={HITSLOP_10}
+                testID="profileHeaderDropdownBtn"
+                style={[
+                  {
+                    flexDirection: 'row',
+                    alignItems: 'center',
+                    justifyContent: 'center',
+                    paddingVertical: 7,
+                    borderRadius: 50,
+                    marginLeft: 6,
+                    paddingHorizontal: 14,
+                  },
+                  pal.btn,
+                ]}>
+                <FontAwesomeIcon
+                  icon="ellipsis"
+                  size={20}
+                  style={t.atoms.text}
+                />
+              </TouchableOpacity>
+            )
+          }}
+        </Menu.Trigger>
+
+        <Menu.Outer>
+          <Menu.Group>
+            <Menu.Item
+              testID="profileShareBtn"
```

Comment:
Was `profileHeaderDropdownShareBtn`, should check if this affects e2e

Reply:
oops, nice find.